requirements-base.txt

```diff
@@ -95,17 +93,7 @@ grpcio>=1.47.0
 # not directly used, but provides a speedup for redis
 hiredis>=0.3.1
 
-# not directly used, but pinned for at least semaphore/symbolic
-cffi>=1.15.0
-
-# not directly used, but pinned for toronado because it doesn't pin these
-cssutils>=2.4.0
-
 # sentry-plugins specific dependencies
 phabricator>=0.7.0
 
-# test dependencies, but unable to move to requirements-test until
-# sentry.utils.pytest and sentry.testutils are moved to tests/
-selenium>=4.1.5
-
```

Comment:
I cleaned the other things up here -- we already pin all transitive dependencies so there's no need to specify them redundantly